content/src/components/editor/rx/workspace.ts

```diff
@@ -122,6 +127,13 @@ export const workspaceHandleRx = Rx.family((workspace: Workspace) =>
         )
       )
 
+      const reset = Rx.fn(() =>
+        Effect.gen(function* () {
+          yield* autosave.resetLocal("save")
+          window.location.reload()
```

Comment:
@titouancreach - this looks really good! The only question I have is whether the reload here is necessary? Just want to make sure I understand the logic before I approve / merge the PR.

Reply:
tbh there is probably another way (better way) to reset the workspace without reloading the page. Reloading the page is probably more "pragmatic" (less side effects maybe). Do you want me to find a way to avoid page reload ?

This code juste remove the localstorage item and reload the page.
Maybe, we can add a timestamp or something because if you reopen the playground 2 weeks after, you probably don't want it to start with your old code

Reply:
Got it